### PR TITLE
Support for python ``3.10``

### PR DESCRIPTION
**Pull request by fselmo**
## What was wrong?

- Python 3.10 needs to have ``PY_SSIZE_T_CLEAN`` defined before including ``Python.h`` (https://docs.python.org/3/c-api/arg.html#strings-and-buffers)

## Solution / changes

- Define ``PY_SSIZE_T_CLEAN`` before including ``Python.h``

### CODE REVIEW

**Comment by kclowes** [comment]
@fjl or @karalabe - can we get someone to take a look at this? We need it to be able to support the latest python versions in `py-evm`. Thank you!

**Review by LefterisJP — COMMENTED** [review]
Is this the only thing for 3.10 support?

Has this been tested?

I can approve but I no longer have merge privileges in this repository.

**Comment by kclowes** [comment]
Thanks @LefterisJP! @fselmo and I just got merge permissions yesterday. I believe he tested it, but will wait for him to confirm before merging. He'll be back on Monday.

**Review by kclowes — APPROVED** [review]

**Comment by fselmo** [comment]
> Thanks @LefterisJP! @fselmo and I just got merge permissions yesterday. I believe he tested it, but will wait for him to confirm before merging. He'll be back on Monday. 

I did test this. We can merge it 👍🏼

**Comment by Mereep** [comment]
Can you push this to pip? Running in this error now, installed it today. Got a version from like 2015.

**Comment by tanmaster** [comment]
> Can you push this to pip? Running in this error now, installed it today. Got a version from like 2015.

I second this.

**Pull request merged**
